src/components/Composer.vue

```diff
@@ -238,31 +248,31 @@
 						</ActionRadio>
 						<ActionRadio :value="dateTomorrowMorning"
 							name="sendLater"
-							:checked="isSendAtTomorrowMorning"
+							:checked="Boolean(isSendAtTomorrowMorning)"
```

Comment:
why does this need a cast and is this change related to the fix?

Reply:
it is not related to this fix, but i've seen this warn in Console ([Vue warn]: Invalid prop: type check failed for prop "checked". Expected Boolean, got Number with value NaN). It is a small this, and i fixed it here. But i can revert this and make another PR.

Reply:
Please fix this separately.

We have to find out why the value ever changes to NaN. Casting NaN to boolean sounds very dangerous. Let's fix this so the property is always true or false.